recipe/build.sh

```diff
@@ -1,6 +1,12 @@
 set -e
 
-./configure --prefix=$PREFIX --enable-official-khronos-headers
+./bootstrap
+
+./configure \
+  --prefix=$PREFIX \
+  --enable-official-khronos-headers \
+  --enable-custom-vendordir=$PREFIX/etc/OpenCL/vendors
```

Comment:
>> Should we create this directory as well?

> Maybe. The only point would be that users can find it in order to manually drop in `.icd` files. Otherwise it'll get created once the first ICD is installed.

PR ( https://github.com/conda-forge/ocl-icd-feedstock/pull/4 ) should help on this front. There is one point we may want to tweak. The rest is pretty straightforward.

>> Also is there a way to still include the system in the search path here?

> That'd be nice. ocl-icd doesn't seem to support multiple search paths when I checked, so it would have to be hacked. For now, users can simply copy the system `.icd` file here and things should just work.

Yeah was thinking about copying. Though a slightly better solution might be adding symlinks. That way if the `.icd`s get removed at the system level (likely due to the associated packages being uninstalled), then this doesn't go looking for non-existent implementations. I suppose we could add this in a post-link step. Though it might confuse users why this doesn't update after a change occurs in the system vendor directory.

It would be great if Linux actually had something like Plan9's union directories. Would make solving this problem trivial. It seems nesting directories doesn't work here either unfortunately as that might be a reasonable hack.